tests/RobotFramework/tests/aws/aws_iot_policy.json

```diff
@@ -0,0 +1,41 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Action": "iot:Connect",
+        "Resource": "arn:aws:iot:us-east-1:227890699892:client/${iot:Connection.Thing.ThingName}"
```

Comment:
The region and client id should not be hardcoded.

Reply:
Actually I work now on a solution not to have a file at all, I want to avoid the need of maintaining a file